ReactWindows/ReactNative.Shared/UIManager/NativeViewHierarchyManager.cs

```diff
@@ -536,6 +567,14 @@ public void ShowPopupMenu(int tag, string[] items, ICallback success)
                     },
                     i));
             }
+#else
+            var menu = new ContextMenu();
+            for (var i = 0; i < items.Length; ++i)
+            {
+                object item = new MenuItem();
+                // TODO: figure out how to add menu items
+            }
```

Comment:
Turns out we just throw NotImplementedException anyway.  I haven't had any requests for this API yet.  You can just delete this code and wrap an #if WINDOWS_UWP around the entire function body except the throw NotImplemented.